cmd/kots/cli/root.go

```diff
@@ -42,6 +42,7 @@ func RootCmd() *cobra.Command {
 	cmd.AddCommand(AppStatusCmd())
 	cmd.AddCommand(GetCmd())
 	cmd.AddCommand(SetCmd())
+	cmd.AddCommand(RunCmd())
```

Comment:
should we make the command hidden or maybe better something like `kots alpha run`?